src/Components/SmartComponents/Reports/DownloadCVEsReport.js

```diff
@@ -63,6 +67,16 @@ const DownloadCVEsReport = ({ filters, params, reportData, buttonProps, isReport
                 allPagesHaveTitle={false}
                 size={'A4'}
                 orientation={'landscape'}
+                footer={
+                    securityRulesPresent && (
+                        <View style={{ marginBottom: 20 }}>
```

Comment:
@leSamo could you create a file that will contain all the styles. it would be better to have them all organized in one place

Reply:
Do you mean global stylesheet or only local for the reports folder?

Reply:
I mean for the reports.

Reply:
I isolated styles inside `Reports/Common/styles.js`.